templates/blog/article_list.html

```diff
@@ -1,15 +1,23 @@
 {% from 'blog/macros.html' import article_list with context %}
 {% extends 'layout.html' %}
 
-{% block title %}Nomad articles - {{ articles[0].title|truncate(40) }}{% endblock %}
+  {% block title %}Nomad articles
```

Comment:
I guess you can wrap all the metas within `{% if articles[0] %}` `{% endif %}`.
If there's no article, there isn't much point of having them anyways.